src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -46,6 +45,17 @@ const propTypes = {
         email: PropTypes.string,
     }),
 
+    /** The transaction associated with the transactionThread */
+    transaction: PropTypes.shape({
```

Comment:
There should be more field here

Reply:
yea, I created transactionPropTypes for this